src/test/java/org/takes/tk/TkProxyTest.java

```diff
@@ -50,30 +51,43 @@
 public final class TkProxyTest {
 
     /**
-     * TkProxy can work.
+     * An array of http methods for testing.
+     */
+    private static final String[] METHODS = {
```

Comment:
@dmzaytsev  I will add another test case for HEAD
And regarding the Connect
```
java.io.IOException: Failed CONNECT request to http://localhost:52991/

	at com.jcabi.http.request.JdkRequest$1.send(JdkRequest.java:127)
	at com.jcabi.http.request.BaseRequest.fetchResponse(BaseRequest.java:336)
	at com.jcabi.http.request.BaseRequest.fetch(BaseRequest.java:251)
	at org.takes.tk.TkProxy.act(TkProxy.java:119)
	at org.takes.tk.TkProxyTest$2.exec(TkProxyTest.java:94)
	at org.takes.http.FtRemote.exec(FtRemote.java:157)
	at org.takes.tk.TkProxyTest.justWorksOnConnect(TkProxyTest.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:234)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:74)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
Caused by: java.net.ProtocolException: Invalid HTTP method: CONNECT
	at java.net.HttpURLConnection.setRequestMethod(HttpURLConnection.java:428)
	at com.jcabi.http.request.JdkRequest$1.send(JdkRequest.java:102)
	... 32 more
```
Should I add a puzzle?